README.md

```diff
@@ -141,6 +169,32 @@ NB: If the `destroy-microbosh` pipeline is run without first cleaning up
 CloudFoundry, it will be necessary to manually clean up the CloudFoundry
 deployment.
 
+### Connecting to your environment
+
+Your PaaS environment is accessed via the cloudfoundry command line tool. This can be
+downloaded from https://github.com/cloudfoundry/cli/releases. The cloudfoundry
+web interface is disabled.
+
+Log into your newly configured environment by running the `cf` command:
+
+```
+cf -a <url of environment> -u admin
+```
+
+The url will be the same that was given to you when you created the deployer-concourse,
+the only difference being the subdomain will be "http://api.yourdomain" instead of
+"http://deployer.yourdomain.
+
+The password will be different to the ones used previously. Ask a friendly colleague
```

Comment:
We should not rely on asking people. 

We should either say that the password is in the `cf-secrets.yml`. Or we should provide a generic way (script or something) to retrieve it.